main.py

```diff
@@ -1,10 +1,98 @@
+import json
+from pathlib import Path
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
+BASE_DIR = Path(__file__).resolve().parent
+
+
+def create_guild(guild: dict):
+    if guild:
+        guild_name = guild["name"]
+        if not Guild.objects.filter(name=guild_name).exists():
+            if guild["description"]:
```

Comment:
I guess this check is not necessary

Reply:
yes, you can just create a guild and the description will be set to null automatically